manager/manager.go

```diff
@@ -687,6 +687,7 @@ func (self *manager) GetFsInfo(label string) ([]v2.FsInfo, error) {
 			Usage:      fs.Usage,
 			Available:  fs.Available,
 			Labels:     labels,
+			InodesFree: fs.InodesFree,
```

Comment:
I think we need to have a way on `FsStats` to know if inodes information is knowable in order to support block level things like devicemapper properly.  We would then only set `InodesFree` if `fs.InodesKnowable` was true for example.  I am not sure how much backwards compatibility concerns there are around `FsStats` struct.